src/color/brand.json

```diff
@@ -478,6 +478,52 @@
           "wcag": "n/a",
           "deprecated": false
         }
+      },
+      "goldgray": {
+        "100": {
+          "value": "C5C1BF",
+          "public": true,
+          "usage": "Tier branding color",
+          "deprecated": false
+        },
+        "200": {
+          "value": "726E6C",
+          "public": true,
+          "usage": "Tier branding color",
+          "deprecated": false
+        }
+      },
+      "gold": {
+        "100": {
+          "value": "CCBC94",
+          "public": true,
+          "usage": "Tier branding color",
+          "deprecated": false
+        },
+        "200": {
+          "value": "7F682E",
+          "public": true,
+          "usage": "Tier branding color",
+          "deprecated": false
+        }
+      },
+      "emerald": {
+        "value": "CCBC94",
```

Comment:
This is a different hash code then what I see in the figma: https://www.figma.com/file/TBMhIMVRMkufKnyxqGzrZO/Color-Explorations?node-id=8%3A1150